### DISCUSSION TITLE
Play mp3 sound effect on repeat until user closes notification

### DISCUSSION THREAD

**Discussion by omnigenous** [dunst-project/dunst#1382] [Q&A]
Just read my week old post: https://github.com/dunst-project/dunst/discussions/1380 and realized it's not clear at all what I was trying to achieve.

I'm making an alarm bash script and I got stuck on one part. I want to play a sound effect (mp3 file) when time is due _on repeat_ until user closes notification manually (I will use `notify-send -t 0 ...`)

Not sure if clarification of my use case helps.

**Comment by bynect** [comment]
Within rules you can specify a script to be run every time the rule is matched by assigning the 'script' option to the name of the script to be run.

you could add a dunstrc rule to match your notif and make a custom script like

```bash
sleep $timeout
play sound ...
```

```ini
[mynotif]
appname = myapp
script = ./sound-script 
```

I thing something along these lines should work. I will close your other issue so we can talk here

**Reply by omnigenous** [reply]
That's where I'm at right now, don't see the benefit of having dunst rule for this instead of just playing the sound inside the script that calls `notify-send`.

But how do I stop playing sound once dunst notification is dismissed manually by the user?
